domain/estimators/logistic.py

```diff
@@ -134,7 +138,8 @@ def train(self, num_epochs=3, batch_size=32):
         :param num_epochs: (int) number of epochs.
         """
         batch_losses = []
-        torch_ds = TensorDataset(self._X, self._Y)
+        X_train, Y_train = self._X.index_select(0, self._train_idx), self._Y.index_select(0, self._train_idx)
```

Comment:
Can we add a small comment here
`# Train only on the examples that do not have NULL init_value`